lms/views/email.py

```diff
@@ -55,33 +55,23 @@ def unsubscribe(self):
 
         return HTTPFound(location=self.request.route_url("email.unsubscribed"))
 
-    @view_config(
-        route_name="email.preferences",
-        request_method="GET",
-        request_param="token",
-    )
-    def preferences_redirect(self):
-        # The token has already been verified by the security policy, so we can
-        # just go right ahead with the redirect.
-        return HTTPFound(
-            location=self.request.route_url("email.preferences"),
-            # Set a cookie to keep the user logged in even though we're
-            # removing the authentication token from the URL.
-            headers=remember(self.request, self.request.authenticated_userid),
-        )
```

Comment:
Removing this `preferences_redirect()` view means that `/email/preferences?token=xyz` requests will now go to the `preferences()` view below.